HISTORY.rst

```diff
@@ -64,6 +64,10 @@
 Unreleased Changes
 ------------------
 
+General
+=======
+* The workbench and the natcap.invest python package now support plugins.
```

Comment:
We should make a note to update this with a little more information and link to the developer documentation.